src/ert/_c_wrappers/enkf/ert_config.py

```diff
@@ -351,28 +388,37 @@ def read_forward_model(
                         for key, val in args:
                             job.private_args.addItem(key, val)
                 except ValueError as err:
-                    raise ConfigValidationError(
-                        errors=f"{err}: 'FORWARD_MODEL {job_name}({args})'\n",
-                        config_file=config_file,
-                    ) from err
+                    combined_error.add_error(
```

Comment:
We will also need to `continue` from this error case, otherwise `job.validate_args` will later create nonsensical error messages.

Reply:
Fixed